doc/api/corepack.md

```diff
@@ -103,13 +103,13 @@ shims aren't currently enabled by default. This has a few consequences:
 npm prevents accidentally overriding the Corepack binaries when doing a global
 install. To avoid this problem, consider one of the following options:
 
-* Don't run this command anymore; Corepack will provide the package manager
+* Don't run this command; Corepack will provide the package manager
   binaries anyway and will ensure that the requested versions are always
-  available, so installing the package managers explicitly isn't needed anymore.
+  available, so installing the package managers explicitly isn't needed.
 
 * Add the `--force` flag to `npm install`; this will tell npm that it's fine to
-  override binaries, but you'll erase the Corepack ones in the process (should
-  that happen, run [`corepack enable`][] again to add them back).
+  override binaries, but you'll erase the Corepack ones in the process. (Run
+  [`corepack enable`][] to add them back.)
```

Comment:
Just a nit. Should we remove the round brackets?

Reply:
I think they're useful here to emphasize that it is an aside/explanation about undoing rather than part of the instructions for the workaround itself. However, I don't feel strongly about that, so if someone disagrees strongly, I'll happily remove them.